### ISSUE TITLE
Missed block proposal duties

### ISSUE THREAD

**Issue by twoeths** [ChainSafe/lodestar#5017]
**Describe the bug**

Missed block proposal 5581154 https://beaconcha.in/epoch/174411

The log said there is no block proposal duties at that epoch

```
grep -e "epoch=174411" -rn validator*.log
validator-2023-01-14.log:44147:Jan-14 12:32:23.107[]                debug: Downloaded SyncDuties epoch=174411, count=0
validator-2023-01-15.log:55640:Jan-15 15:44:23.207[]                debug: Downloaded attester duties epoch=174411, dependentRoot=0x1f468914b7bdbd89f03bbab196cdcdc5cea566e66bbf08b3f87c55cc8e0c6954, count=64
validator-2023-01-15.log:56001:Jan-15 15:50:47.055[]                debug: Downloaded SyncDuties epoch=174411, count=0
validator-2023-01-15.log:56005:Jan-15 15:50:47.073[]                debug: Downloaded proposer duties epoch=174411, dependentRoot=0xd35b7d58a95969ce56069c2ca1cfecc665b99b5a3bc38c4886dce35a4f0e02af, count=0
validator-2023-01-15.log:56006:Jan-15 15:50:47.096[]                debug: Downloaded attester duties epoch=174411, dependentRoot=0x1f468914b7bdbd89f03bbab196cdcdc5cea566e66bbf08b3f87c55cc8e0c6954, count=64
validator-2023-01-15.log:56025:Jan-15 15:50:59.055[]                debug: Downloaded proposer duties epoch=174411, dependentRoot=0xd35b7d58a95969ce56069c2ca1cfecc665b99b5a3bc38c4886dce35a4f0e02af, count=0
validator-2023-01-15.log:56037:Jan-15 15:51:11.056[]                debug: Downloaded proposer duties epoch=174411, dependentRoot=0xd35b7d58a95969ce56069c2ca1cfecc665b99b5a3bc38c4886dce35a4f0e02af, count=0
validator-2023-01-15.log:56047:Jan-15 15:51:23.054[]                debug: Downloaded proposer duties epoch=174411, dependentRoot=0xd35b7d58a95969ce56069c2ca1cfecc665b99b5a3bc38c4886dce35a4f0e02af, count=0
validator-2023-01-15.log:56058:Jan-15 15:51:35.054[]                debug: Downloaded proposer duties epoch=174411, dependentRoot=0xd35b7d58a95969ce56069c2ca1cfecc665b99b5a3bc38c4886dce35a4f0e02af, count=0
validator-2023-01-15.log:56078:Jan-15 15:51:47.232[]                debug: Downloaded proposer duties epoch=174411, dependentRoot=0xd35b7d58a95969ce56069c2ca1cfecc665b99b5a3bc38c4886dce35a4f0e02af, count=0
validator-2023-01-15.log:56092:Jan-15 15:51:59.055[]                debug: Downloaded proposer duties epoch=174411, dependentRoot=0xd35b7d58a95969ce56069c2ca1cfecc665b99b5a3bc38c4886dce35a4f0e02af, count=0
validator-2023-01-15.log:56109:Jan-15 15:52:11.055[]                debug: Downloaded proposer duties epoch=174411, dependentRoot=0xd35b7d58a95969ce56069c2ca1cfecc665b99b5a3bc38c4886dce35a4f0e02af, count=0
validator-2023-01-15.log:56119:Jan-15 15:52:23.054[]                debug: Downloaded proposer duties epoch=174411, dependentRoot=0xd35b7d58a95969ce56069c2ca1cfecc665b99b5a3bc38c4886dce35a4f0e02af, count=0
validator-2023-01-15.log:56137:Jan-15 15:52:35.060[]                debug: Downloaded proposer duties epoch=174411, dependentRoot=0xd35b7d58a95969ce56069c2ca1cfecc665b99b5a3bc38c4886dce35a4f0e02af, count=0
validator-2023-01-15.log:56150:Jan-15 15:52:47.053[]                debug: Downloaded proposer duties epoch=174411, dependentRoot=0xd35b7d58a95969ce56069c2ca1cfecc665b99b5a3bc38c4886dce35a4f0e02af, count=0
validator-2023-01-15.log:56160:Jan-15 15:52:59.060[]                debug: Downloaded proposer duties epoch=174411, dependentRoot=0xd35b7d58a95969ce56069c2ca1cfecc665b99b5a3bc38c4886dce35a4f0e02af, count=0
validator-2023-01-15.log:56172:Jan-15 15:53:11.054[]                debug: Downloaded proposer duties epoch=174411, dependentRoot=0xd35b7d58a95969ce56069c2ca1cfecc665b99b5a3bc38c4886dce35a4f0e02af, count=0
validator-2023-01-15.log:56185:Jan-15 15:53:23.054[]                debug: Downloaded proposer duties epoch=174411, dependentRoot=0xd35b7d58a95969ce56069c2ca1cfecc665b99b5a3bc38c4886dce35a4f0e02af, count=0
validator-2023-01-15.log:56195:Jan-15 15:53:35.054[]                debug: Downloaded proposer duties epoch=174411, dependentRoot=0xd35b7d58a95969ce56069c2ca1cfecc665b99b5a3bc38c4886dce35a4f0e02af, count=0
validator-2023-01-15.log:56207:Jan-15 15:53:47.055[]                debug: Downloaded proposer duties epoch=174411, dependentRoot=0xd35b7d58a95969ce56069c2ca1cfecc665b99b5a3bc38c4886dce35a4f0e02af, count=0
validator-2023-01-15.log:56217:Jan-15 15:53:59.054[]                debug: Downloaded proposer duties epoch=174411, dependentRoot=0xd35b7d58a95969ce56069c2ca1cfecc665b99b5a3bc38c4886dce35a4f0e02af, count=0
validator-2023-01-15.log:56221:Jan-15 15:54:11.056[]                debug: Downloaded proposer duties epoch=174411, dependentRoot=0xd35b7d58a95969ce56069c2ca1cfecc665b99b5a3bc38c4886dce35a4f0e02af, count=0
validator-2023-01-15.log:56240:Jan-15 15:54:23.052[]                debug: Downloaded proposer duties epoch=174411, dependentRoot=0xd35b7d58a95969ce56069c2ca1cfecc665b99b5a3bc38c4886dce35a4f0e02af, count=0
validator-2023-01-15.log:56251:Jan-15 15:54:35.056[]                debug: Downloaded proposer duties epoch=174411, dependentRoot=0xd35b7d58a95969ce56069c2ca1cfecc665b99b5a3bc38c4886dce35a4f0e02af, count=0
validator-2023-01-15.log:56262:Jan-15 15:54:47.055[]                debug: Downloaded proposer duties epoch=174411, dependentRoot=0xd35b7d58a95969ce56069c2ca1cfecc665b99b5a3bc38c4886dce35a4f0e02af, count=0
validator-2023-01-15.log:56272:Jan-15 15:54:59.057[]                debug: Downloaded proposer duties epoch=174411, dependentRoot=0xd35b7d58a95969ce56069c2ca1cfecc665b99b5a3bc38c4886dce35a4f0e02af, count=0
validator-2023-01-15.log:56284:Jan-15 15:55:11.054[]                debug: Downloaded proposer duties epoch=174411, dependentRoot=0xd35b7d58a95969ce56069c2ca1cfecc665b99b5a3bc38c4886dce35a4f0e02af, count=0
validator-2023-01-15.log:56294:Jan-15 15:55:23.056[]                debug: Downloaded proposer duties epoch=174411, dependentRoot=0xd35b7d58a95969ce56069c2ca1cfecc665b99b5a3bc38c4886dce35a4f0e02af, count=0
validator-2023-01-15.log:56304:Jan-15 15:55:35.055[]                debug: Downloaded proposer duties epoch=174411, dependentRoot=0xd35b7d58a95969ce56069c2ca1cfecc665b99b5a3bc38c4886dce35a4f0e02af, count=0
validator-2023-01-15.log:56315:Jan-15 15:55:47.055[]                debug: Downloaded proposer duties epoch=174411, dependentRoot=0xd35b7d58a95969ce56069c2ca1cfecc665b99b5a3bc38c4886dce35a4f0e02af, count=0
validator-2023-01-15.log:56319:Jan-15 15:55:59.058[]                debug: Downloaded proposer duties epoch=174411, dependentRoot=0xd35b7d58a95969ce56069c2ca1cfecc665b99b5a3bc38c4886dce35a4f0e02af, count=0
validator-2023-01-15.log:56323:Jan-15 15:56:11.054[]                debug: Downloaded proposer duties epoch=174411, dependentRoot=0xd35b7d58a95969ce56069c2ca1cfecc665b99b5a3bc38c4886dce35a4f0e02af, count=0
validator-2023-01-15.log:56334:Jan-15 15:56:23.054[]                debug: Downloaded proposer duties epoch=174411, dependentRoot=0xd35b7d58a95969ce56069c2ca1cfecc665b99b5a3bc38c4886dce35a4f0e02af, count=0
validator-2023-01-15.log:56345:Jan-15 15:56:35.055[]                debug: Downloaded proposer duties epoch=174411, dependentRoot=0xd35b7d58a95969ce56069c2ca1cfecc665b99b5a3bc38c4886dce35a4f0e02af, count=0
validator-2023-01-15.log:56355:Jan-15 15:56:47.059[]                debug: Downloaded proposer duties epoch=174411, dependentRoot=0xd35b7d58a95969ce56069c2ca1cfecc665b99b5a3bc38c4886dce35a4f0e02af, count=0
validator-2023-01-15.log:56368:Jan-15 15:56:59.055[]                debug: Downloaded proposer duties epoch=174411, dependentRoot=0xd35b7d58a95969ce56069c2ca1cfecc665b99b5a3bc38c4886dce35a4f0e02af, count=0
```

Metrics didn't show any block production error in the last 2 days
<img width="771" alt="Screen Shot 2023-01-17 at 08 40 38" src="https://user-images.githubusercontent.com/10568965/212791209-b5d4c9e9-3f84-45bc-bb5b-22f5cf3109a7.png">

**Expected behavior**

Should not miss block production duties

**Comment by twoeths** [comment]
validator log

```
Jan-15 15:51:00.337[]              ^[[36mverbose^[[39m: Found new chain head slot=5581153, head=0x7380275ad813672f0640fe06f2cf88bb58dead5819bbb3e001dc0950f425b23c, previousDuty=0x1f468914b7bdbd89f03bbab196cdcdc5cea566e66bbf08b3f87c55cc8e0c6954, currentDuty=0xd35b7d58a95969ce56069c2ca1cfecc665b99b5a3bc38c4886dce35a4f0e02af
Jan-15 15:51:00.337[]                ^[[34mdebug^[[39m: HttpClient request routeId=produceAttestationData
Jan-15 15:51:00.390[]                ^[[34mdebug^[[39m: HttpClient response routeId=produceAttestationData
Jan-15 15:51:00.392[]                ^[[34mdebug^[[39m: Signed attestation slot=5581153, index=27, head=0x7380275ad813672f0640fe06f2cf88bb58dead5819bbb3e001dc0950f425b23c, validatorIndex=270084
Jan-15 15:51:00.393[]                ^[[34mdebug^[[39m: Signed attestation slot=5581153, index=51, head=0x7380275ad813672f0640fe06f2cf88bb58dead5819bbb3e001dc0950f425b23c, validatorIndex=270102
Jan-15 15:51:00.394[]                ^[[34mdebug^[[39m: Signed attestation slot=5581153, index=27, head=0x7380275ad813672f0640fe06f2cf88bb58dead5819bbb3e001dc0950f425b23c, validatorIndex=270126
Jan-15 15:51:00.395[]                ^[[34mdebug^[[39m: HttpClient request routeId=submitPoolAttestations
Jan-15 15:51:00.513[]                ^[[34mdebug^[[39m: HttpClient response routeId=submitPoolAttestations
Jan-15 15:51:00.514[]                 ^[[32minfo^[[39m: Published attestations slot=5581153, count=3
Jan-15 15:51:11.001[]                ^[[34mdebug^[[39m: HttpClient request routeId=getProposerDuties
Jan-15 15:51:11.055[]                ^[[34mdebug^[[39m: HttpClient response routeId=getProposerDuties
Jan-15 15:51:11.056[]                ^[[34mdebug^[[39m: Downloaded proposer duties epoch=174411, dependentRoot=0xd35b7d58a95969ce56069c2ca1cfecc665b99b5a3bc38c4886dce35a4f0e02af, count=0
Jan-15 15:51:15.001[]                ^[[34mdebug^[[39m: HttpClient request routeId=produceAttestationData
Jan-15 15:51:15.055[]                ^[[34mdebug^[[39m: HttpClient response routeId=produceAttestationData
Jan-15 15:51:15.059[]                ^[[34mdebug^[[39m: Signed attestation slot=5581154, index=3, head=0x7380275ad813672f0640fe06f2cf88bb58dead5819bbb3e001dc0950f425b23c, validatorIndex=270094
Jan-15 15:51:15.062[]                ^[[34mdebug^[[39m: Signed attestation slot=5581154, index=28, head=0x7380275ad813672f0640fe06f2cf88bb58dead5819bbb3e001dc0950f425b23c, validatorIndex=270122
Jan-15 15:51:15.062[]                ^[[34mdebug^[[39m: HttpClient request routeId=submitPoolAttestations
Jan-15 15:51:15.158[]                ^[[34mdebug^[[39m: HttpClient response routeId=submitPoolAttestations
Jan-15 15:51:15.158[]                 ^[[32minfo^[[39m: Published attestations slot=5581154, count=2
Jan-15 15:51:23.000[]                ^[[34mdebug^[[39m: HttpClient request routeId=getProposerDuties
Jan-15 15:51:23.053[]                ^[[34mdebug^[[39m: HttpClient response routeId=getProposerDuties
Jan-15 15:51:23.054[]                ^[[34mdebug^[[39m: Downloaded proposer duties epoch=174411, dependentRoot=0xd35b7d58a95969ce56069c2ca1cfecc665b99b5a3bc38c4886dce35a4f0e02af, count=0
Jan-15 15:51:25.060[]              ^[[36mverbose^[[39m: Found new chain head slot=5581155, head=0x77a974e088add6e0c33f5b59fe94a12a7b2d3180765f3165a84f4c05313bfa02, previousDuty=0x1f468914b7bdbd89f03bbab196cdcdc5cea566e66bbf08b3f87c55cc8e0c6954, currentDuty=0xd35b7d58a95969ce56069c2ca1cfecc665b99b5a3bc38c4886dce35a4f0e02af
```

**Comment by twoeths** [comment]
looked into wrong node, should open a separate issue
